package.json

```diff
@@ -51,6 +51,7 @@
     "jsdom-global": "3.0.2",
     "mocha": "5.2.0",
     "nyc": "15.0.0",
+    "sinon": "^9.0.2",
```

Comment:
use fixed versions as the rest of dependencies

Reply:
done.